app/shared/productTypes.tsx

```diff
@@ -31,6 +31,15 @@ export type ProductFriendlyName =
   | "digital subscription"
   | "Guardian Weekly subscription"
   | "subscription";
+export type ProductShortenedFriendlyName =
+  | "membership"
+  | "recurring contribution" // TODO use payment frequency instead of 'recurring' e.g. monthly annual etc
+  | "newspaper"
+  | "newspaper voucher"
+  | "home delivery"
+  | "digital"
+  | "Guardian Weekly"
+  | "subscription";
```

Comment:
this type definition need only include the ones you're specifying below, and tbh probably needn't exist if we're not restricting the type at the point it's used (just adds noise/confusion)
_To clarify, keep the `shortenedFriendlyName` property of `ProductType` just don't think it needs at type_